eng/init-detect-binaries.proj

```diff
@@ -4,21 +4,33 @@
     <TargetFramework>$(NetCurrent)</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(ToolsDir)BinaryToolKit\BinaryToolKit.csproj" />
+  </ItemGroup>
+
+  <UsingTask TaskName="BinaryToolTask" AssemblyFile="$(BinaryToolKitAssembly)" TaskFactory="TaskHostFactory" />
+
+  <PropertyGroup>
+    <BinariesMode Condition="'$(BinariesMode)' == ''">Validate</BinariesMode>
+    <AllowedBinariesFile Condition="'$(AllowedBinariesFile)' == ''">$(RepoRoot)eng/allowed-vmr-binaries.txt</AllowedBinariesFile>
+  </PropertyGroup>
+
   <Target Name="DetectBinaries"
           BeforeTargets="Build"
           Condition="'$(SkipDetectBinaries)' != 'true'">
```

Comment:
If you don't want reviews yet, just let me know :)

I would move this Condition out to Build.proj where this project is referenced so that BinaryToolKit.csproj doesn't even get built.